builds/jazz-build-module/sls-app/serverless-build-rules.yml

```diff
@@ -353,9 +353,10 @@ provider:
 
   iamManagedPolicies: # Optional IAM Managed Policies, which allows to include the policies into IAM Role
     sbr-type: "[arn-iam-policy]"
-    sbr-render: merge
+    sbr-render: user-only
```

Comment:
In future if we decide to add some managed policy in config, does `merge` work as expected i.e. combine the config list with the user list? Should test with multiple config policies and multiple policies in user yml.

Reply:
Here are a couple of policy ARN to whitelist so you can test it since u need at least a few more: arn:aws:iam::484695107796:policy/iam_lambda
arn:aws:iam::484695107796:policy/Lambda_Execution

Reply:
Added sbr-render as merge. Also tested with more policies. Working as expected.